GitDepend/Commands/DryUpdateCommand.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO.Abstractions;
+using System.Linq;
+using GitDepend.Busi;
+using GitDepend.CommandLine;
+using GitDepend.Resources;
+using GitDepend.Visitors;
+using NuGet;
+
+namespace GitDepend.Commands
+{
+	/// <summary>
+	/// An implementation of <see cref="ICommand"/> that recursively does a dry run update for all dependencies.
+	/// </summary>
+	public class DryUpdateCommand : ICommand
```

Comment:
I would prefer to have this be an extra option on the existing update command.

Something like `dep update --dry`